CHANGES.rst

```diff
@@ -2,7 +2,7 @@
 Changelog
 =========
 
-Plateau 4.1.2 (2022-10-20)
+Plateau 4.1.2 (2022-10-24)
```

Comment:
4.1.2 is already released. Can you add an entry for 4.1.3? Then we would be ready for a merge.

Reply:
Done.